### DISCUSSION TITLE
Licensing

### DISCUSSION THREAD

**Discussion by fiscdupenn** [vercel/next.js#68035] [Help]
### Summary

Getting our organization licensed correctly.

### Additional information

```js
We are a small organization that is starting a new project with Next.js/Vercel.  Who do I talk with to be sure the licensing and organization is setup correctly?
```

**Comment by icyJoseph** [comment]
Isn't Next.js MIT? https://github.com/vercel/next.js/blob/canary/license.md so your legal department has to use that information... For every tool you use, at least check if there's a LICENSE.

**Comment by fiscdupenn** [comment]
I am talking about setting up a primary Vercel billing account and getting
everyone in the user team licensed correctly.

**Reply by icyJoseph** [reply]
Aha, please reach Vercel Support (I think from your dashboard, you should be able to find it)
